### ISSUE TITLE
Resistor not used in sample?

### ISSUE THREAD

**Issue by aviks** [jakkaj/Xamling-IOT#2]
Hi @jakkaj nice library and writeups, thanks. 

On the circuit diagram in  `Samples/HC-SR04`, the 650 ohm resistor doesn't seem to connected to anything? Is that an oversight?

**Comment by jakkaj** [comment]
Cheers :) The repo is quite old now, not quite sure what I was thinking at the time :/ Are things... erm... still working?

**Comment by aviks** [comment]
no, that circuit that does not work. Still trying to debug it. :)

**Comment by jakkaj** [comment]
Sorry :/ I don't think I even have the components lying around. If you do fix, please leave a note and I can fix up the sample.
